keps/sig-api-machinery/4355-coordinated-leader-election/README.md

```diff
@@ -469,45 +453,42 @@ type LeaseSpec struct {
 For the identity leases, a new lease will be created
 
 ```go
-type IdentityLeaseSpec struct {
+type LeaseCandidateSpec struct {
+  // The fields BinaryVersion and CompatibilityVersion will be mandatory labels instead of fields in the spec
 	// BinaryVersion indicates the binary version of the component
-	BinaryVersion string
-
+	// BinaryVersion string
 	// Compatibility Version indicates the compatibility version of the component. Must be less than or equal to the binary version
-	CompatibilityVersion string
+	// CompatibilityVersion string
```

Comment:
remove

Reply:
I've commented these out but kept them for now. Joe has some thoughts around keeping part of it that I'll let him comment.